recipes/spalter/meta.yaml

```diff
@@ -1,25 +1,22 @@
 {% set version = "0.2.2" %}
-{% set name = "spalter" %}
 {% set sha256 = "eb6acf379286b8b678a5d988292c6dac9d6f797b4d1cf8ac9840f6b34bc6473a" %}
 
 package:
-  name: {{ name }}
+  name: spalter
   version: {{ version }}
 
 build:
   number: 0
-  skip: True # [osx]
+  skip: True  # [osx]
 
 source:
-  fn: {{ name }}-{{version}}.tar.gz
-  url: https://bitbucket.org/tillux/{{ name }}/get/v{{ version }}.tar.gz
+  url: https://bitbucket.org/tillux/spalter/get/v{{ version }}.tar.gz
   sha256: {{ sha256 }}
 
 requirements:
   build:
+    - {{ compiler('c') }}
```

Comment:
So, this `- {{ compiler('c') }}` in `requirements/build` does some "magic" for us:
- Depending on the build configuration (i.e., just OS for Bioconda currently), that Jinja2 template evaluates to the compiler package for the given language (`'c'`), e.g., `toolchain_c_linux-64` (and likely `gcc_linux-64` once we update to newer compilers).
- That `toolchain_c_linux-64` does the rest of the magic then:
  - It has (via another package called `toolchain`) some "activation script" that automatically sets a bunch of compiler/linker variables (`LDFLAGS`, etc.).
  - It has a `run_export` definition for the `libgcc-ng` package which then automagically gets added to `requirements/run` to `spalter`'s recipe.

At some point we may (or rather will, I assume) add `{{ compiler('rust') }}`, too (via the `conda-forge-pinning` package). It's just not there yet :wink:.

Reply:
Thanks for the elaboration, will be useful for other rust-related recipes!